gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/PersistentOplogSet.java

```diff
@@ -484,19 +494,37 @@ private void findInconsistencyInternal(Map<String, List<VdrBucketId>> oneSet, fi
         if (currlist.size() > numbuckets_of_root) {
           currlist.forEach(x -> {
             if (!bucketIdsOfRoot.contains(x.bucketId)) {
-              badBuckets.add(x);
-              if (!this.inconsistent) {
-                this.inconsistent = true;
+              String badBucketName = x.vdr.getName();
+              String[] parts = SystemProperties.SHADOW_SCHEMA_NAME_WITH_SEPARATOR.split("_");
+              if (badBucketName != null && badBucketName.indexOf(parts[0]) != -1 &&
+                  badBucketName.indexOf(parts[1]) != -1) {
```

Comment:
might be better to use something like: PartitionedRegionHelper.getPRPath(badBucketName).endsWith(SystemProperties.SHADOW_TABLE_SUFFIX)

Reply:
Well not exactly endsWith can be used but contains can be used. endsWith cannot be there as bucket ids are there at the end. I am leaving like this and checking in. Let me know if you have a strong reason to change this. I will change later.